app/store/chatList.go

```diff
@@ -300,6 +302,51 @@ func (s *Sessions) CreateSessionForE164(tel string, UUID string) *Session {
 	return ses
 }
 
+func (s *Sessions) CreateSessionForUUID(UUID string) *Session {
+	contact := GetContactForUUID(UUID)
+	ses := &Session{
+		Tel:          contact.Tel,
+		Name:         contact.Name,
+		Active:       true,
+		IsGroup:      false,
+		Notification: true,
+		UUID:         UUID,
+	}
+	if s.Len == 0 {
+		ses.ID = 1
+	}
+	ses, err := SaveSession(ses)
+	log.Errorln("[axolotl] CreateSessionForUUID failed:", ses.ID)
+
+	if err != nil {
+		log.Errorln("[axolotl] CreateSessionForUUID failed:", err)
+		return nil
+	}
+	s.Sess = append(s.Sess, ses)
+	s.Len = len(s.Sess)
+
+	message := &Message{
+		Message:    "Chat created",
+		SID:        ses.ID,
+		ChatID:     ses.Tel,
```

Comment:
Why is the chatId still the phone?

Reply:
The ChatID is a legacy value and can be removed in future releases. For now i keep it consistent through the code because I don't know where it's still used.